installer/Migration/V5_4_0/Migration.php

```diff
@@ -148,6 +153,56 @@ public function up(): void
             $this->getSchemaHelper()->addForeignKey('ohrm_claim_request', $foreignKeyConstraint3);
         }
     }
+
+    /**
+     * @return void
+     */
+    private function modifyClaimEventTables()
+    {
+        $this->getSchemaHelper()->addOrChangeColumns('ohrm_claim_event', [
+            'is_deleted' => ['Type' => Type::getType(Types::BOOLEAN), 'Notnull' => true, 'Default' => 0],
+            'status' => ['Type' => Type::getType(Types::BOOLEAN), 'Notnull' => false, 'Default' => null,'CustomSchemaOptions' => ['collation' => null,'charset' => null]],
+        ]);
+    }
+
+    /**
+     * @return void
+     */
+    private function convertStatusOldValuesToBoolean($table): void
+    {
+        $count = $this->getTableRecordCount($table);
+        $batchSize = 10;
+        for ($i=0; $i<$count; $i+=$batchSize) {
+            $result = $this->createQueryBuilder()
+                    ->select('id', 'status')
+                    ->from($table)
+                    ->setFirstResult($i)
+                    ->setMaxResults($batchSize)
+                    ->executeQuery();
+            foreach ($result->fetchAllAssociative() as $row) {
+                $this->createQueryBuilder()
+                        ->update($table)
+                        ->set('status', ':status')
+                        ->where('id = :id')
+                        ->setParameter('id', $row['id'])
+                        ->setParameter('status', $row['status']=='on' ? 1 : 0)
+                        ->executeStatement();
```

Comment:
need to check this twise

Reply:
implement according to the changeClaimExpenseTypeTableStatusToBoolean after rebasing